testkit/src/main/scala/org/bitcoins/testkit/wallet/BitcoinSWalletTest.scala

```diff
@@ -86,13 +103,22 @@ trait BitcoinSWalletTest
 
   def withNewWalletAndBitcoind(test: OneArgAsyncTest): FutureOutcome = {
     val builder: () => Future[WalletWithBitcoind] = composeBuildersAndWrap(
-      createNewWallet,
+      createNewWallet _,
```

Comment:
Is this kind of syntax going to be ok with scala 2.11?

Reply:
It compiles fine with sbt and Scala version set to 2.11.12